File: 485-max-consecutive-ones/485-max-consecutive-ones.cs
public class Solution {
    public int FindMaxConsecutiveOnes(int[] nums) {
        
        
        int maxLen = 0;
        
        int counter = 0;
        
        for( var i = 0; i < nums.Length ; i ++)
        {
            if(nums[i] == 0)
            {
                counter = 0;
            }else
            {
                counter++;
                maxLen = Math.Max(maxLen,counter);
            }
        }
            
        
        return maxLen;
    }
}

File: 485-max-consecutive-ones/485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        
                if (nums == null) {
            return -1;
        }

        // variables

        int len = nums.length;

        int max = Integer.MIN_VALUE;

        int counter = 0;


        for(int i = 0; i < len; i++)
        {
            if (nums[i] == 1) {
                counter++;
            }else
            {
                counter = 0;
            }
            max = Math.max(max, counter);
        }
        return max;

}
    
}